ReactAndroid/src/main/java/com/facebook/react/modules/image/ImageLoaderModule.java

```diff
@@ -45,6 +48,48 @@ public String getName() {
     return "ImageLoader";
   }
 
+  @ReactMethod
+  public void getSize(
+      String uriString,
+      final Callback successCallback,
+      final Callback errorCallback) {
+    Uri uri = Uri.parse(uriString);
+    ImageRequest request = ImageRequestBuilder.newBuilderWithSource(uri).build();
+
+    DataSource<CloseableReference<CloseableImage>> dataSource =
+      Fresco.getImagePipeline().fetchDecodedImage(request, mCallerContext);
+
+    DataSubscriber<CloseableReference<CloseableImage>> dataSubscriber =
+      new BaseDataSubscriber<CloseableReference<CloseableImage>>() {
+        @Override
+        protected void onNewResultImpl(
```

Comment:
You're right, after some logging with isClosed() I've also added `image.close()`